python/ml4ir/applications/ranking/features/feature_fns/robust_scaler_transform.py

```diff
@@ -0,0 +1,61 @@
+import tensorflow as tf
+
+from ml4ir.applications.ranking.model.layers.rank_transform import RobustScalerLayer
+from ml4ir.base.features.feature_fns.base import BaseFeatureLayerOp
+from ml4ir.base.io.file_io import FileIO
+
+
+class RobustScaler(BaseFeatureLayerOp):
```

Comment:
@mohazahran You need to also update it under https://github.com/salesforce/ml4ir/blob/master/python/ml4ir/base/features/feature_layer.py#L41